package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-services-stream",
-  "version": "8.7.0",
+  "version": "8.7.1",
```

Comment:
Revert to 8.7.0. I see that there has been no 8.7.0 release --> https://github.com/mojaloop/central-services-stream/releases